### PR TITLE
Update spec editors

### PR DESCRIPTION
**Pull request by chrisn**
This PR add Jean-Yves and Will as co-editors, and move Chris, Mounir, and Vi (who is also no longer active in the WG) to former editors. 

See https://www.w3.org/2022/11/08-mediawg-minutes.html#t07 and https://lists.w3.org/Archives/Public/public-media-wg/2022Nov/0004.html

@willcassella @jyavenard We just need to add your w3cids to complete this PR.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/pull/200.html" title="Last updated on Nov 17, 2022, 9:40 AM UTC (3875d1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/media-capabilities/200/5793c9b...3875d1d.html" title="Last updated on Nov 17, 2022, 9:40 AM UTC (3875d1d)">Diff</a>

### CODE REVIEW

**Comment by jyavenard** [comment]
I'm not sure what a w3cid is, but my username login on w3.org is `jyavenard`

File: index.bs [hunk 1]
```diff
@@ -7,9 +7,12 @@ TR: https://www.w3.org/TR/media-capabilities/
 Shortname: media-capabilities
 Level: None
 Group: mediawg
-Editor: Mounir Lamouri, w3cid 45389, Google Inc. https://www.google.com/
-Editor: Chris Cunningham, w3cid 114832, Google Inc. https://www.google.com/
-Editor: Vi Nguyen, w3cid 116349, Microsoft Corporation https://www.microsoft.com/
+Editor: Jean-Yves Avenard, w3cid xxx, Apple Inc. https://www.apple.com/
```
Comment (dontcallmedom) [inline index.bs:10]:
```suggestion
Editor: Jean-Yves Avenard, w3cid 115886, Apple Inc. https://www.apple.com/
```

File: index.bs [hunk 2]
```diff
@@ -7,9 +7,12 @@ TR: https://www.w3.org/TR/media-capabilities/
...
+Editor: Will Cassella, w3cid xxx, Google Inc. https://www.google.com/
```
Comment (dontcallmedom) [inline index.bs:11]:
```suggestion
Editor: Will Cassella, w3cid 139598, Google Inc. https://www.google.com/
```

**Comment by tidoust** [comment]
> I'm not sure what a w3cid is, but my username login on w3.org is `jyavenard`

One day, we may be able to use the username instead of the somewhat internal w3cid to identify an author in authoring tools. Until then, to find your w3cid, you need to look at the address bar in the [my profile](https://www.w3.org/users/myprofile) page.

Your w3cid is 115886 and Will's w3cid is 139598.

**Comment by chrisn** [comment]
Thank you @dontcallmedom and @tidoust!

**Pull request merged**